webpack.config.js

```diff
@@ -230,27 +251,33 @@ module.exports = (env, args = {}) => {
   };
 
   const rules = [
-    {
-      test: /\.js$/,
-      exclude: /core-js/,
-      use: {
-        loader: 'babel-loader',
-      },
-    },
     {
       test: /\.vue$/,
       loader: 'vue-loader',
       options: {
         hotReload,
-        compilerOptions: { // @see https://github.com/vuejs/vue/tree/dev/packages/vue-template-compiler#options
+        compilerOptions: { // @see https://vue-loader.vuejs.org/options.html
           whitespace: 'condense',
-          directives: {
-            // Remove the Pimcore directive from production.
-            'pimcore': () => !isProduction,
-            'pimcore-areabrick': () => !isProduction,
-            'pimcore-snippet': () => !isProduction,
-            'pimcore-template': () => !isProduction,
-          }
+        }
+      },
+    },
+    {
+      test: /\.tsx?$/,
+      loader: 'ts-loader',
+      exclude: /node_modules/,
+      options: {
+        // disable type checker - we will use it in fork plugin
```

Comment:
What do you mean with `...in fork plugin`?

Reply:
cleared up the comment